docs/source/developing/releasing.adoc

```diff
@@ -12,15 +12,23 @@ The following sections describe how to release Minishift.
 [[release-prereqs]]
 == Prerequisites
 
-You must have a https://help.github.com/articles/creating-an-access-token-for-command-line-use[GitHub personal access token]
+* You must have a https://help.github.com/articles/creating-an-access-token-for-command-line-use[GitHub personal access token]
 defined in your environment as _GITHUB_ACCESS_TOKEN_.
+* You need _gnutar_ installed on your machine. See also GitHub issue link:https://github.com/minishift/minishift/issues/657[#657].
+
+== Preparing the GitHub milestone
+
+. Verify the milestone you want to release
+ * Move unresolved issues into the next milestone
```

Comment:
I would say let's use the term Open/Closed issues instead of 'Resolve/Unresolve' which seems to be term of Jira.

![screenshot from 2017-04-05 19-04-06](https://cloud.githubusercontent.com/assets/1132451/24707887/e4426e30-1a32-11e7-94a9-e02fff109248.png)

then, this and below lines would be:
* Move open issues into the next milestone
* Review closed issues and make sure they are classified correctly